packages/dialtone-css/lib/build/less/components/input.less

```diff
@@ -270,18 +270,15 @@
 
 //  $$  VALIDATION STATES
 //  ----------------------------------------------------------------------------
-.d-input--warning,
-.d-textarea--warning {
```

Comment:
no reason to have these d-textarea--state classes I think? They do exactly the same thing as d-input--state

Reply:
Seems like it, but would that break anything else on product?

Reply:
possible, but it's part of the migration above so should be fine after it's been done.